packages/my-queries-storage/src/hooks.ts

```diff
@@ -0,0 +1,76 @@
+import { useEffect } from 'react';
+import { useSyncExternalStore } from 'use-sync-external-store/shim';
+import type { FavoriteQuerySchema, RecentQuerySchema } from './query-storage';
+import { FavoriteQueryStorage, RecentQueryStorage } from './query-storage';
+
+let lastId = 0;
+const subscriptions = new Map<number, () => void>();
```

Comment:
This is just a `Set`? We don't use the key in any meaningful way?

Reply:
You are right. I don't know why I used Map here, maybe just leftovers.